modules/client/examples.md

```diff
@@ -575,7 +586,7 @@ contextPlugin.setFull(contextParams);
 console.log(contextPlugin)
 ```
 
-### Create a TokenVoting client
+### Create an TokenVoting client
```

Comment:
This is introducing a regression. Mathias fixed this here, but probably it needs fixing on the source files as well?